### ISSUE TITLE
VSCode formatter

### ISSUE THREAD

**Issue by tomislavhren** [wix/stylable-intelligence#319]
How to enable formatting in vscode without having to associate `.st.css` to CSS (in that case, I lose the benefits of the stylable intelligence)?

**Comment by tomrav** [comment]
Sadly we are missing this capability at the moment, I will try and take a look and see if I can wire things together quickly-ish.

Until then, perhaps a less satisfying answer is that you can install the `Beautify` extension and add the following snippet to your VSCode config. This should make it work for Stylable files. 

```
    "beautify.language": {
        "css": [
            "css",
            "less",
            "scss",
            "stylable"
        ]
    },
```

**Comment by tomrav** [comment]
We've just published version `v0.6.1` of the `Stylable-Intelligence`, it includes formatting capabilities without requiring any additional extensions or configuration.

Closing the issue as it is resolved.
